src/main/java/buildcraftAdditions/compat/buildcraft/recipe/BCRecipeManager.java

```diff
@@ -176,7 +176,7 @@ public static void addBCRecipes() {
 			GameRegistry.addRecipe(new ItemStack(ItemsAndBlocks.itemSorter), "SUS", "PCP", "SGS", 'S', Blocks.stone, 'U', EnumMachineUpgrades.AUTO_OUTPUT.getItemStack(), 'P', BCItems.PIPE_ITEMS_LAPIS, 'C', Blocks.chest, 'G', BCItems.PIPE_ITEMS_GOLD);
 
 		if (ConfigurationHandler.enabled("ColoringTool"))
-			GameRegistry.addRecipe(new ItemStack(ItemsAndBlocks.pipeColoringTool), "  S", " S ", "W  ", 'W', new ItemStack(Blocks.wool, 1, OreDictionary.WILDCARD_VALUE), 'S', Items.stick);
+			GameRegistry.addRecipe(new ItemStack(ItemsAndBlocks.pipeColoringTool), "  S", " C ", "W  ", 'W', new ItemStack(Blocks.wool, 1, OreDictionary.WILDCARD_VALUE), 'S', Items.stick, 'C', ItemsAndBlocks.powerCapsuleTier1);
```

Comment:
I did this In my previous PR so it would make sense that the tool was using power